src/js/nav/globalNav.js

```diff
@@ -25,9 +25,9 @@ export const grouppedNav = {
                 title: 'Settings',
                 subItems: [
                     {
-                        id: 'webhooks',
-                        title: 'Webhooks',
-                        reload: 'webhooks'
+                        id: 'hooks',
+                        title: 'Hooks',
+                        reload: 'hooks'
```

Comment:
To follow the pattern for Settings apps, this should probably be `reload: 'settings/hooks'`.